go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/coredns/coredns
 
-go 1.23.0
+go 1.24.0
```

Comment:
We follow upstream Go's release process. Go supports the last two Go releases, which we want to follow here.